Sources/SpeziHealthKit/Sample Types/SampleType+QuantityTypes.swift

```diff
@@ -0,0 +1,66 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import HealthKit
+
+
+extension HealthKitSampleType where Sample == HKQuantitySample {
+    /// The sample type representing step count quantity samples
+    public static var stepCount: Self { .quantity(.stepCount, displayTitle: "Step Count", displayUnit: .count()) }
+    
+    /// The sample type representing blood oxygen saturation quantity samples
+    public static var bloodOxygen: Self { .quantity(.oxygenSaturation, displayTitle: "Blood Oxygen", displayUnit: .percent()) }
+    
+    /// The sample type representing heart rate quantity samples
+    public static var heartRate: Self { .quantity(
+        .heartRate,
+        displayTitle: "Heart Rate",
+        displayUnit: .count() / .minute()
+//        expectedValuesRange: 60...150
```

Comment:
I like the idea of directly injecting the expected ranges here.

Reply:
ideally, i'd have the chart simply look at the min/max of the values it's trying to plot and auto-adjust its y axis based on that.

the issue is that this is actually only sometimes what you want: if you e.g. have heart rate samples and some of them exceed the "normal" range of heart rates, you want the chart to adjust to that. but if you e.g. only have heart rates in the range `90...94`, you don't want to get a chart with a y axis that only covers a range of like 10 BPM.

i'll keep and properly implement the `expectedValuesRange` approach